<launch>
  <node type="laser_scan_assembler" pkg="laser_assembler" name="my_assembler">
    <remap from="scan" to="scan"/>
    <param name="max_scans" type="int" value="1" />
    <param name="fixed_frame" type="string" value="laser_base" />
  </node>
  

</launch>
